superset/connectors/sqla/views.py

```diff
@@ -144,7 +144,7 @@ class TableColumnInlineView(
         "extra": utils.markdown(
             "Extra data to specify column metadata. Currently supports "
             'certification data of the format: `{ "certification": "certified_by": '
-            '"Taylor Swift", "details": "This column is the source of truth." '
+            '["Taylor Swift", "Harry Styles"], "details": "This column is the source of truth." '
```

Comment:
We should also add a test that confirms backward-compatibility (the a single value is passed instead of an array).